### PR TITLE
Qa8 1481

### PR DESCRIPTION
**Pull request by natdem**
Add TC-066

### CODE REVIEW

File: data/expected.json [hunk 1]
```diff
@@ -34,5 +34,7 @@
   "nameField70SymbolsError": "70 symbols max",
   "requiredMessage": "Required",
   "TryAgainTxt": "Try again!",
-  "fieldIsEmpty": ""
+  "fieldIsEmpty": "",
+  "agePlaceholder":"Hero's age"
```
Comment (isenkovska) [inline data/expected.json:38]: can you please move agePlacholder into row 12, just under age label;

File: data/selectors.js [hunk 2]
```diff
@@ -27,7 +27,8 @@ const elements = {
     "imgAlt": "img[alt]",
     "submit": ".ant-btn",
     "story": ".card-text",
-    "tryAgain": ".btn-primary"
+    "tryAgain": ".btn-primary",
```
Comment (isenkovska) [inline data/selectors.js:30]: could you please delete comma from  the row 30, after "tryAgain": ".btn-primary",

**Review by isenkovska — CHANGES_REQUESTED** [review]

File: data/expected.json (hunk 1 above)
Comment (natdem) [reply data/expected.json]: All fixed

File: data/selectors.js (hunk 2 above)
Comment (natdem) [reply data/selectors.js]: All fixed

**Review by isenkovska — APPROVED** [review]
looks good

**Review by ivansh83 — APPROVED** [review]
Good job!

**Pull request merged**